src/The_Cartridge_Header.md

```diff
@@ -127,145 +137,160 @@ Code | Publisher
 `99` | Pack in soft
 `A4` | Konami (Yu-Gi-Oh!)
 
-## 0146 - SGB Flag
+## 0146 – SGB Flag
 
-Specifies whether the game supports SGB functions, common values are:
+This byte specifies whether the game supports SGB functions.
```

Comment:
Why repeat "this byte"? we're explaining it anyway..

Reply:
I agree the repetition isn't all that great, but I think it's acceptable to be more precise, and can be less monotonous by alternative with "this area" and/or "this field".

Reply:
I think in a technical text precision is preferable to elegance, and I simply thought that "this byte" is a consistent way to format things.